Fix a typo in description of span sharing (#37)
openzipkin/b3-propagation: README.md

The most common propagation use case is to copy a trace context from a client 
sending an RPC request to a server receiving it.

> In this case, the same span IDs are used, which means that both the client and

Reviewer: can you add a unit test?

just kidding

Reviewer: ooops sorry I think we should grammar

```suggestion
In this case, the same span ID is used, which means that both the client and
```
